src/app/integrations/intacct/intacct.component.ts

```
@@ -77,13 +80,27 @@ export class IntacctComponent implements OnInit {
+          () => this.getOrCreateWorkspace()
+        );
+      } else {
+        this.getOrCreateWorkspace();
+      }
+    });
+  }
+
   ngOnInit(): void {
     this.router.events.subscribe((event) => {
       if (event instanceof NavigationEnd) {
         (window as any).Appcues && (window as any).Appcues.page();
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)